### PR TITLE
Vegu2021

### PR DESCRIPTION
**Pull request by DarshikaManral**
Please review the page for vegu presentations. At present its not linked to the main website, but its possible to navigate to UtrechtTeam page from here.

### CODE REVIEW

File: utrechtteam_vegu2021.html [hunk 1]
```diff
@@ -0,0 +1,265 @@
+<!DOCTYPE html>
```
Comment (erikvansebille) [inline utrechtteam_vegu2021.html:1]: Why not simply name this file vegu21.html? Then the url that we can share is oceanparcels.org/vegu21 (so without the 'utrechtteam_')

File: utrechtteam_vegu2021.html [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+        </div>
+    </div>
+</nav>
+
+<!-- Page Content -->
+<div class="container">
+
+    <hr>
+
+    <h1 class="display-4" style="text-align: center;">Ocean Parcels team at vEGU2021<a href="https://www.egu21.eu/">
```
Comment (erikvansebille) [inline utrechtteam_vegu2021.html:73]: make the 'vEGU21' text also a link?

File: utrechtteam_vegu2021.html [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+        </div>
+
+        <div class="col-lg-3 col-md-4 col-sm-6 mb-3">
+            <div class="card h-100 text-center">
+                <img class="card-img-top team-photo" src="images/DelphineLobelle.jpg" alt="">
+                <div class="card-body">
+                    <h4 class="card-title">Delphine Lobelle</h4>
+                    <p class="card-subtitle mb-2 text-muted">Merel Kooi, Albert A. Koelmans, Charlotte Laufkotter, Cleo E. Jongedijk, Christian Kehl, and Erik van Sebille</p>
+                    <br>
+                    <h5 class="card-subtitle mb-2 text">Mon, 26 Apr, 11:07–11:09</h5>
```
Comment (erikvansebille) [inline utrechtteam_vegu2021.html:111]: Would it work better to put these dates in the footer, so that they are all aligned?

**Review by erikvansebille — COMMENTED** [review]
Hi @DarshikaManral. Looks great! A few minor comments below

**Review by daanreijnders — APPROVED** [review]
Looks great! Can't think of any improvements besides the ones Erik already proposed

**Review by SLYpma — APPROVED** [review]
Looks great!

**Review by pdnooteboom — APPROVED** [review]
Looks very nice!

**Review by erikvansebille — APPROVED** [review]

**Pull request merged**